test/events.test.js

```diff
@@ -0,0 +1,251 @@
+      .post('/admin/events')
+      .set('Authorization', `bearer ${token}`)
+      .send(event);
+
+    expect(response.status).to.equal(200)
+    expect(response.body).to.be.a('object')
+    expect(response.body.message).to.be.a('string')
+  })
+
+
+
+  //test to check for the editing of events 
+  it("should be able to edit events",async()=>{
+    
+    let event = await eventModel.findOne({name:"webkriti"})
+    
+    /*SUCCESS*/
+    let response = await request(Server)
+      .put(`/admin/events/${event._id}`)
+      .set('Authorization', `bearer ${token}`)
+      .send({name:"gitkriti"});
+    
+    expect(response.status).to.equal(200)
+    expect(response.body.message).to.be.a('string')
+    
+    /*FAILURE*/
+    response = await request(Server)
+      .put(`/admin/events/615f758ed92d2b5ee1975eb8`) ///usiing a random event id
+      .set('Authorization', `bearer ${token}`)
+      .send({name:"gitkriti"});
+
+    expect(response.status).to.equal(400)
+    expect(response.body.message).to.be.a('string')
+  })
+
+
+
+
+  //Test to delete events 
+  it("should delete events",async()=>{
+
+    let event = await eventModel.findOne({name:"webkriti"})
+
+    let response = await request(Server)
+      .delete(`/admin/events/${event._id}`)
+      .set('Authorization', `bearer ${token}`)
+      .send({name:"gitkriti"});
+    
+    expect(response.status).to.equal(200)
+    expect(response.body.message).to.be.a('string')
+
+  })
+
+  
+  
+  //Test to get the QR code of events
+  it("should get the QR code of events",async ()=>{
+
+    let event = await eventModel.findOne({name:"webkriti"})
+
+    /*SUCCESS*/
+    let response = await request(Server)
+      .get(`/admin/events/qr/${event._id}/1`)
+      .set('Authorization', `bearer ${token}`);
+    
+    expect(response.status).to.equal(200)
+    expect(response.body.message).to.be.a('string')
+    expect(response.body.qr).to.be.a('string')
+
+    /*FAILURE*/
+    response = await request(Server)
+      .get(`/admin/events/qr/${event._id}/5`) //Invalid number of days
+      .set('Authorization', `bearer ${token}`);
+    
+    expect(response.status).to.equal(400)
+    expect(response.body.message).to.be.a('string')
+    })
+
+  
+  
+  //To add goodies to the events for the eligible students 
+  it("should allow to add goodies",async () => {
+
+    const event = await eventModel.findOne({name:"webkriti"})
+
+    await userModel.create({
+      _id:"2020BCS-044",
+      name:"Kailash Kejriwal",
+      password:"123",
+      roll:"2020BCS-044"
+    })
+
+    /*SUCCESS*/
+    let response = await request(Server)
+      .post(`/admin/events/goodies`)
+      .set('Authorization', `bearer ${token}`)
+      .send({roll:"2020BCS-044",eventId:event._id});
+
+      expect(response.status).to.equal(200)
+      expect(response.body.message).to.be.a('string')
+      
+      /*FAILURE*/
+    response = await request(Server)
+      .post(`/admin/events/goodies`) //Invalid number of days
```

Comment:
I guess the comment `Invalid number of days` isn't applicable here. Same for the following test.